experimental/packages/exporter-trace-otlp-proto/package.json

```diff
@@ -5,14 +5,21 @@
   "main": "build/src/index.js",
```

Comment:
I am not familiar with these. am coming from the browser world and these refer to node.  I checked a few other packages and noticed that not all of them have `module` and `esnext` fields and the esm/esnext files in the `files` field. For an example see `exporter-trace-otlp-grpc`. If this is a concern, I think this can be taken up in separate PR making these changes in all the packages.

Reply:
@pichlermarc I added these entry points and also made the changes to build the esm and esnext files. I have tested the fetch-proto example in opentelemetry-web and it works fine. Please review when you get a chance and thanks for raising this topic.

Reply:
Looks great, thank you for addressing this! :slightly_smiling_face: